docs/source/miscellaneous/dropwizard-metrics.rst

```diff
@@ -0,0 +1,61 @@
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.meanBorrowWaitTimeMillis``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.meanIdleTimeMillis``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.proportionDestroyedByBorrower``
+ - ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPoolingContainer.<host>.proportionDestroyedByEvictor``
+
+**Histograms**
+
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.byteSizeTx``
+
+**Meters**
+
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requestConnectionExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requestExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.<host>.requests``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requestConnectionExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requestExceptions``
+- ``com.palantir.atlasdb.keyvalue.cassandra.CassandraClientPool.requests``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskReadOnly.failures``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskReadOnly.failures.java.lang.IllegalArgumentException``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskWithRetry.failures``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskWithRetry.failures.feign.FeignException``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskWithRetry.failures.java.lang.IllegalArgumentException``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskWithRetry.failures.java.lang.IllegalStateException``
+
+**Timers**
+
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.getLockService``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskReadOnly``
+- ``com.palantir.atlasdb.transaction.api.LockAwareTransactionManager.runTaskWithRetry``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitAcquireLocks``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitCheckingForConflicts``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitPutCommitTs``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitTotalTimeSinceTxCreation``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.commitWrite``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.getRows``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.millisForPunch``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.processedRangeMillis``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.transactionMillis``
+- ``com.palantir.atlasdb.transaction.impl.SnapshotTransaction.waitForCommitTsMillis``
```

Comment:
I backed these Transaction ones out due to overhead on useTables which is invoked a ton but all local, but we should timers for each KeyValueService method.